libs/service-portal/finance/src/lib/paths.ts

```diff
@@ -1,6 +1,6 @@
 export enum FinancePaths {
   FinanceRoot = '/fjarmal',
-  FinanceStatus = '/fjarmal/stada',
+  FinanceStatus = 'fjarmal/stada',
```

Comment:
why is this the only one without a leading slash?

Reply:
I have no idea. I'll fix it 🤦